src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -146,9 +160,19 @@
                constants/content-type-album
                [album/album-message message-data context on-long-press]
 
-               [not-implemented/not-implemented [content.unknown/unknown-content message-data]])
+               [not-implemented/not-implemented
+                [content.unknown/unknown-content message-data]])
+
              (when @show-delivery-state?
-               [status/status outgoing-status])])]]]))))
+               [status/status outgoing-status])])]
+          (when show-reactions?
```

Comment:
could you pls elaborate on why reactions were moved here?

Reply:
Because of the layout. You can see it when you for example press on a message — the whole message needs to be highlighted in this case, not just the message without the reactions area.